### PR TITLE
Updated mon db trimming to psi only and added slow ops

### PR DESCRIPTION
**Pull request by tintumathew10**
Signed-off-by: tintumathew10 <tmathew@redhat.com>

Updated mon db trimming to psi only and added slow ops

### CODE REVIEW

File: pipeline/metadata/5.3.yaml [hunk 1]
```diff
@@ -402,14 +402,29 @@
...
+    - stage-2
+
 - name: "RADOS regression testing for Mon DB trimming"
   suite: "suites/pacific/rados/tier-3_rados_test-mon-db-trimming.yaml"
   global-conf: "conf/pacific/rados/11-node-cluster.yaml"
   platform: "rhel-8"
   rhbuild: "5.3"
   inventory:
     openstack: "conf/inventory/rhel-8-latest.yaml"
-    ibmc: "conf/inventory/ibm-vpc-rhel-8-latest.yaml"
```
Comment (psathyan) [inline pipeline/metadata/5.3.yaml:412]: In additional to this removal, you would need to remove the tags `ibmc, openstack` and add the tag `openstack-only`

**Review by psathyan — COMMENTED** [review]
Do we need to modify 6.0 file also?

File: pipeline/metadata/5.3.yaml (hunk 1 above)
Comment (tintumathew10) [reply pipeline/metadata/5.3.yaml:412]: done

**Comment by tintumathew10** [comment]
> Do we need to modify 6.0 file also?

yes, I will ceate another PR for the same.

**Review by psathyan — APPROVED** [review]

**Pull request merged**